app/components/UnderlinedLink.tsx

```diff
@@ -1,12 +1,17 @@
 import * as React from "react";
-
 import * as styles from "./UnderlinedLink.scss";
 
 interface IUnderlinedLinkProps {
   href: string;
+  className?: string;
 }
 
-export const UnderlinedLink: React.SFC<IUnderlinedLinkProps> = ({ href, children }) =>
-  <a href={href} className={styles.underlinedLink}>
+export const UnderlinedLink: React.SFC<IUnderlinedLinkProps> = ({ href, className, children }) =>
+  <a
+    href={href}
+    className={
+      className !== undefined ? `${styles.underlinedLink} ${className}` : styles.underlinedLink
```

Comment:
use `classnames` modules. It's already dependency in a project. You can write it simply like: `cn(styles.underlinedLink, className)` and it will handle the rest